src/lang/en-US.ts

```diff
@@ -900,23 +900,27 @@ const Docs: ResolvedIntlConfig['messages'] = {
 };
 
 const NewTransform: ResolvedIntlConfig['messages'] = {
-    'newTransform.modal.title': `Build a new collection`,
+    'newTransform.modal.title': `Build a new transformation`,
     'newTransform.language.title': `Language`,
     'newTransform.language.sql': `SQL`,
     'newTransform.language.ts': `Typescript`,
-    'newTransform.collection.label': `Collection Name`,
+    'newTransform.collection.label': `Transformation Name`,
     'newTransform.errors.collection': `Select A Source Collection`,
     'newTransform.errors.name': `Name Your Transform`,
     'newTransform.errors.prefixMissing': `No prefix selected`,
     'newTransform.errors.namePattern': `Name does not match pattern`,
     'newTransform.errors.nameInvalid': `Invalid entity name`,
     'newTransform.errors.nameMissing': `Missing entity name`,
-    'newTransform.errors.gitPod': `Failed to open GitPod`,
-    'newTransform.stepper.step1.label': `Select your collection`,
+    'newTransform.errors.urlNotGenerated': `We failed to generate the proper URL to start GitPod. ${Error['error.tryAgain']}`,
+    'newTransform.errors.gitPodWindow': `Failed to open GitPod. Your browser may be blocking it from opening. Please ensure your browser allows pop-ups.`,
```

Comment:
New errors match the ones made for OAuth... just with `oauth` replaced with `GitPod`